src/db/migrations/20230618104332_feature_flags.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ export function up (knex) {
+  return knex.schema
+    .createTable('feature_flags', table => {
+      table.string('name').primary().unique()
+      table.boolean('is_enabled').defaultTo(false)
+      table.text('description').default('')
+      table.specificType('dependencies', 'character varying(255)[]')
+      table.specificType('allow_list', 'character varying(255)[]')
+      table.specificType('wait_list', 'character varying(255)[]')
+      table.timestamp('created_at').defaultTo(knex.fn.now())
+      table.timestamp('modified_at').defaultTo(knex.fn.now())
+      table.timestamp('expired_at')
+      table.string('owner')
```

Comment:
Just wondering if it would be desirable to potentially have multiple owners?

Also, not sure how you both feel about this (it might just be noisy), but I sometimes feel like it can be useful to explicitly mark every column as `nullable` or `notNullable`, just so the former jumps out more.

Reply:
Hmm.. I think it's usually best practice to have one. In the future, we CAN add the functionality to transfer ownership.